pipeline/outputs/dash0.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Send logs to Dash0
+---
+
+# Dynatrace
+
+Stream logs to [Dash0](https://www.dash0.com) by utilizing the **opentelemetry** plugin to send data to the Dash0 log ingress.
+
+## Configuration parameters
+
+| Key                        | Description | Default |
+| -------------------------- | ----------- | ------- |
+| `header`                   | The specific header for bearer authorization, where {your-Auth-token-here} is your Dash0 Auth Token. | Authorization Bearer {your-Auth-token-here} |
+| `host`                     | Your Dash0 ingress endpoint. | `ingress.eu-west-1.aws.dash0.com` |
+| `port`                     | TCP port of your Dash0 ingress endpoint. | `443` |
+| `metrics_uri`              | Specify an optional HTTP URI for the target web server listening for metrics | `/v1/metrics` |
+| `logs_uri`                 | Specify an optional HTTP URI for the target web server listening for logs | `/v1/logs` |
+| `traces_uri`               | Specify an optional HTTP URI for the target web server listening for traces | `/v1/traces`  |
+| `tls`                      | Specify to use TLS. | `on` |
```

Comment:
We can probably skip this and just reference the transport security section as these are common to all.